test/e2e/density.go

```diff
@@ -181,15 +187,13 @@ var _ = Describe("Density", func() {
 		// TODO: Expose runLatencyTest as ginkgo flag.
 		{podsPerNode: 3, runLatencyTest: false, interval: 10 * time.Second},
 		{podsPerNode: 30, runLatencyTest: true, interval: 10 * time.Second},
-		// More than 30 pods per node is outside our v1.0 goals.
-		// We might want to enable those tests in the future.
 		{podsPerNode: 50, runLatencyTest: false, interval: 10 * time.Second},
-		{podsPerNode: 100, runLatencyTest: false, interval: 1 * time.Second},
+		{podsPerNode: 100, runLatencyTest: true, interval: 10 * time.Second},
```

Comment:
I'm almost convinved this will not work, because IIUC we have a hard limit on MaxPods set to 100, and there are also some system pods, and additionally, we are starting some pods as part of "latency" phase. So I think this should be something less than 100 (like 95?)